### ISSUE TITLE
first time user bugs

### ISSUE THREAD

**Issue by fcollman** [seung-lab/chunkflow#178]
I'm trying to use this for the first time on a cluster using the latest image, and my chunkflow command is returning the following error

`Starting worker 1
Traceback (most recent call last):
  File "/usr/local/bin/chunkflow", line 11, in <module>
    load_entry_point('chunkflow', 'console_scripts', 'chunkflow')()
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 1290, in invoke
    return _process_result(rv)
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 1224, in _process_result
    value = ctx.invoke(self.result_callback, value, **ctx.params)
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/root/workspace/chunkflow/chunkflow/flow/flow.py", line 95, in process_commands
    stream = operator(stream)
  File "/root/workspace/chunkflow/chunkflow/flow/flow.py", line 108, in operator
    return func(stream, *args, **kwargs)
  File "/root/workspace/chunkflow/chunkflow/flow/flow.py", line 622, in delete_task_in_queue
    for task in tasks:
  File "/root/workspace/chunkflow/chunkflow/flow/flow.py", line 402, in cloud_watch
    for task in tasks:
  File "/root/workspace/chunkflow/chunkflow/flow/flow.py", line 1269, in save
    for task in tasks:
  File "/root/workspace/chunkflow/chunkflow/flow/flow.py", line 1118, in crop_margin
    for task in tasks:
  File "/root/workspace/chunkflow/chunkflow/flow/flow.py", line 1024, in inference
    verbose=state['verbose']) as inferencer:
  File "/root/workspace/chunkflow/chunkflow/chunk/image/convnet/inferencer.py", line 95, in __init__
    assert (self.input_size is not None) or (self.patch_num is not None)
AssertionError`

Can you provide any insight as to what is going wrong?

Here is the command I am running.
`seq "$PROCESS_NUM" | parallel -j "$PROCESS_NUM" --delay 300 --ungroup echo Starting worker {}\; chunkflow --verbose 1 --mip $MIP_LEVEL \
          fetch-task --queue-name="$QUEUE_NAME" --visibility-timeout=$VISIBILITY_TIMEOUT \
          cutout --volume-path="$IMAGE_LAYER_PATH" --expand-margin-size 16 64 64 --name cutout-em \
          inference \
          --convnet-model="/nets/pytorch-3dunet/net_allen.py" \
          --convnet-weight-path="/nets/${CONVNET_FILE_NAME}" \
          --input-patch-size 256 256 256 --output-patch-overlap 32 32 32 \
          --num-output-channels 1 --framework='pytorch' --batch-size 1 \
          crop-margin save --name="save-netoutput" --volume-path="$OUTPUT_LAYER_PATH" \
          --upload-log cloud-watch delete-task-in-queue;`

**Comment by fcollman** [comment]
I figured this out.. I needed to set --patch-num.  I believe this should be changed to be a mandatory argument as i don't see input-size as being an argument.

Also, the documentation for setup-env should say explicitly that the outputs of this need to be transferred into the chunk-flow calls.  I believe its output could be formatted/engineered more nicely to facilitate this process... for example printing arguments in the format that the click command line interface wants for example

--patch-num 7 11 11 

rather than

patch_num (7, 11, 11)
